ubiquity/dell-bootstrap.py

```diff
@@ -1251,4 +1251,8 @@ def install(self, target, progress, *args, **kwargs):
         install_misc.record_installed(to_install)
         install_misc.record_removed(to_remove)
 
+        #copy the to_install package list into /tmp as backup to check
+        apt_installed = "/var/lib/ubiquity/apt-installed"
+        shutil.copy(apt_installed, os.path.join(self.target, 'tmp', os.path.basename(apt_installed)))
+
```

Comment:
The /target/var/lib/ubiquity path does not exist at that moment when is running dell-bootstrap.py. So it unavoidably needs to put somewhere else, not the direct path.